src/MSBuild/XMake.cs

```diff
@@ -4502,5 +4510,26 @@ private static void ShowVersion()
                 Console.Write(ProjectCollection.Version.ToString());
             }
         }
+
+        private static void ShowFeatureAvailability(string[] features)
+        {
+            if (features.Length == 1)
+            {
+                string featureName = features[0];
+                FeatureStatus availability = Features.CheckFeatureAvailability(featureName);
+                Console.WriteLine(availability);
+            }
+            else
+            {
+                var jsonNode = new JsonObject();
+                foreach (string featureName in features)
+                {
+                    jsonNode[featureName] = Features.CheckFeatureAvailability(featureName).ToString();
+                }
+
+                var s_options = new JsonSerializerOptions() { AllowTrailingCommas = false, WriteIndented = true };
```

Comment:
nit: Naming convention, should be `options` without the `s_` prefix. Or the `ToJsonString` call can be moved to `JsonOutputFormatter` so it can use the `s_options` static field there.